sherpa-onnx/csrc/sherpa-onnx-keyword-spotter.cc

```diff
@@ -0,0 +1,122 @@
+// sherpa-onnx/csrc/sherpa-onnx-keyword-spotter.cc
```

Comment:
Please also add a CI test for this file, including test cases both for C++ and Python.